packages/terra-notification-dialog/themes/orion-fusion-theme/orion-fusion-theme.scss

```diff
@@ -1,40 +1,14 @@
 @import '~terra-mixins/lib/Mixins';
 
 :global {
-  --terra-notification-dialog-background: #fff;
-  --terra-notification-dialog-border-radius: 5px;
-  --terra-notification-dialog-small-horizontal-inset: 10px;
-  --terra-notification-dialog-body-background-color: #fff;
-  --terra-notification-dialog-body-padding: 1.25rem;
   --terra-notification-dialog-header-background-color: #f9f9f9;
   --terra-notification-dialog-header-border-bottom: 0.0625rem solid #dedfe0;
   --terra-notification-dialog-header-font-size: 1.125rem;
```

Comment:
Do we need to keep these around? If an old version needs to use these, will they get aggregated & pulled in by that version?

Reply:
You don't need to keep old styles around. They're bound to the component version now.

Reply:
removed: https://github.com/cerner/terra-framework/pull/1105/commits/9605d29d15b2d90959460b61343fa7b54fd4ee30